docs/angular/your-first-app/5-adding-mobile.md

```diff
@@ -93,8 +93,8 @@ Next, head back over to the `loadSaved()` function we implemented for the web ea
 ```tsx
 public async loadSaved() {
   // Retrieve cached photo array data
-  const photoList = await Preferences.get({ key: this.PHOTO_STORAGE });
-  this.photos = JSON.parse(photoList.value) || [];
+  const { value } = await Preferences.get({ key: this.PHOTO_STORAGE });
+  this.photos = (value ? JSON.parse(value) : []);
```

Comment:
Let's keep the line consistent with the other packages by adding the type to the end:
```tsx
this.photos = (value ? JSON.parse(value) : []) as UserPhoto[];
```